api/routers/agentic_rag.py

```
@@ -73,28 +73,32 @@
         if not rag_system.use_agentic_rag:
-            raise HTTPException(status_code=HTTP_SERVICE_UNAVAILABLE,
-                detail='Code example search requires agentic RAG to be enabled'
-                )
-        example_query = f'Show me code examples for: {request.topic}'
-        result = await rag_system.query_documentation(query=example_query,
-            source_filter=request.framework, query_type='code_examples',
-            max_results=5)
-        return StandardResponse(success=True, data=result, message=
-            f'Found {len(result.sources)} relevant code examples')
+            raise HTTPException(
+                status_code=HTTP_SERVICE_UNAVAILABLE, detail="Code example search requires agentic RAG to be enabled"
```

Pylint (3.3.8)

[error] Undefined variable 'HTTP_SERVICE_UNAVAILABLE'

(E0602)